scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
> can't stop a not-cross-compilation-aware configure script setting things up wrong ;-)

Yeah, if you have some native deps that you build outside of node-gyp...

Ok, I stopped promoting electron-builder docker image :) (currently, I don't have time to test and add ia32 version of image).